src/inputs/UploadFile/UploadFile.js

```diff
@@ -24,40 +24,45 @@ export const UploadFile = (props) => {
     file,
     editMode,
     isFileValid,
+    error,
     labels,
     tooltipText,
     isDirty,
   } = props;
+  if (isFileValid) {
+    console.warn('"isFileValid" prop is deprecated in UploadFile. Please consider removing this prop.');
```

Comment:
Can you add the suggestion to add the `error` prop instead, in the warning message?

Reply:
I have hesitated to add it...